### ISSUE TITLE
Getting "The "--filter" option does not exist error" randomly.

### ISSUE THREAD

**Issue by nikunjkotecha** [davidtrainer/blt-site-studio#17]
```
[Acquia\Blt\Robo\Tasks\DrushTask] Running /var/www/html/***/vendor/bin/drush @self pm:list --filter="cohesion_sync" --status=Enabled --field=status --no-interaction -v in /var/www/html/***/docroot

In DrushArgvInput.php line 227:
                                                          
  [Symfony\Component\Console\Exception\RuntimeException]  
  The "--filter" option does not exist.                   
```                                             

Platform: Acquia ACP
Site Studio: 6.5
Drupal: 9.1

It is single site and we do `blt drupal:install -v --no-interaction -D drush.ansi=false` in post-code-update hook where we get this error.

**Comment by davidtrainer** [comment]
@nikunjkotecha what version of drush are you using?

**Comment by mikemadison13** [comment]
this is fixed by https://github.com/davidtrainer/blt-site-studio/pull/22

**Comment by mikemadison13** [comment]
TLDR though, doing a drush cc drush usually makes the --filter option show up.
